musicbot/entry.py

```diff
@@ -328,7 +356,10 @@ def __init__(self, playlist, url, title, *, destination=None, **meta):
         self.url = url
         self.title = title
         self.destination = destination
-        self.duration = 0
+        self.duration = None
+        log.info('Cannot estimate duration for stream, estimated time of the queue will not be available'
+                 'until this stream got removed from the queue.'
+                 'stream name: {}'.format(self.title))
```

Comment:
We didn't log this before, no reason to log it now.

Reply:
If you say so then I will do it that way. I was just being a little pedantic that if some commands show that duration cannot be estimated and there's nothing mentioned then some users might come to ask us why.